packages/lesswrong/components/users/UsersMenu.jsx

```diff
@@ -79,16 +79,16 @@ class UsersMenu extends PureComponent {
         >
             {showNewButtons &&
               <MenuItem onClick={()=>openDialog({componentName:"NewQuestionDialog"})}>
-                Ask Question [Beta]
+                Ask Question
```

Comment:
I'm not opposed to this change per se, but I don't like unrelated things sneaking into PRs like this.